incubator/demo-ycsb/files/ycsb_gen.sh

```diff
@@ -75,24 +75,14 @@ if [ -n "${LB_POLICY}" ]; then
     LOADBALANCER_POLICY="&LBPolicy=${LB_POLICY}"
 fi
 
-. /etc/profile.d/cloud.sh
-
-: ${POD_NAME:=$(hostname)}
-: ${NODE_NAME:=$(cloud_hostname)}
-: ${NODE_ZONE:=$(cloud_zone)}
-
-LBVAR_POD_NAME="&pod=${POD_NAME}"
-LBVAR_NODE_NAME="&node=${NODE_NAME}"
-LBVAR_NODE_ZONE="&zone=${NODE_ZONE}"
-
 #check for nuo.properties file overrides
 if [ "$PEER_ADDRESS" != "" ] && [ "$DB_NAME" != "" ]; then
     AGENT_DB_NAME_VALUE="-p db.url=jdbc:com.nuodb://${PEER_ADDRESS}/${DB_NAME}?schema=${db_schema}"
 
     if [ "$TE_DIRECT" == "true" ]; then
         AGENT_DB_NAME_VALUE="${AGENT_DB_NAME_VALUE}&direct=true"
     else
-        AGENT_DB_NAME_VALUE="${AGENT_DB_NAME_VALUE}${LOADBALANCER_QUERY}${LOADBALANCER_POLICY}${LBVAR_POD_NAME}${LBVAR_NODE_NAME}${LBVAR_NODE_ZONE}"
+        AGENT_DB_NAME_VALUE="${AGENT_DB_NAME_VALUE}${LOADBALANCER_QUERY}${LOADBALANCER_POLICY}"
```

Comment:
So does this mean we cannot pass any variable on the URL to represent which ZONE (region) the YCBS client is running in - or any other "locality"?

Joe used to rely on this for his demo - can his demo now be modified to still show load-balancing across ZONE/region?

Reply:
the TEs are not labeled. So using labels to select them can not work.

What this code was doing, is that it was passing down some strings, nothing would match them, so it would fall back to `ANY`.

Reply:
So what you're saying is that this was already broken?

Reply:
yes.

> We removed the corresponding cloud.sh auto-tagging from the database pods in Helm Chart release 2.0.0 and as such, keeping it in YCSB was serving no purpose.

Reply:
So do we have any plan going forward to support ZONE and/or region load-balancing using the New Admin LB system?

- I can see a pretty easy way to support it using k8s balancers, and direct=true.